mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb

```diff
@@ -0,0 +1,20 @@
+resource "google_project" "my_project" {
+  provider   = google-beta
+  name       = "tf-test-project"
+  project_id = "<%= ctx[:vars]['project_id'] %>"
+  org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+  auto_create_network = false
+}
+
+resource "google_service_usage_consumer_quota_override" "override" {
+  provider       = google-beta
+  dimensions = {
+    region = "us-central1"
+  }
+  project        = google_project.my_project.project_id
+  service        = "compute.googleapis.com"
+  metric         = "compute.googleapis.com%2Fn2_cpus"
```

Comment:
Would the forward slash character not work here?

Reply:
Haven't tried, let me check

Reply:
It doesn't work.


```
=== CONT  TestAccServiceUsageConsumerQuotaOverride_regionConsumerQuotaOverrideExample
2021/02/25 15:43:54 [WARN] Got error running Terraform: 
Error: Invalid escape sequence

  on ../../../../../../../../var/folders/b1/dthn83bs2qbcrg38qszm22440000gn/T/plugintest318460841/work555631348/config721254248/terraform_plugin_test.tf line 17, in resource "google_service_usage_consumer_quota_override" "override":
  17:   limit          = "\project\region"

The symbol "p" is not a valid escape sequence selector.

    provider_test.go:270: Step 1/1 error: Error running pre-apply refresh: 
        Error: Invalid escape sequence
        
          on ../../../../../../../../var/folders/b1/dthn83bs2qbcrg38qszm22440000gn/T/plugintest318460841/work555631348/config721254248/terraform_plugin_test.tf line 17, in resource "google_service_usage_consumer_quota_override" "override":
          17:   limit          = "\project\region"
        
        The symbol "p" is not a valid escape sequence selector.
        
--- FAIL: TestAccServiceUsageConsumerQuotaOverride_regionConsumerQuotaOverrideExample (14.29s)
FAIL
FAIL	github.com/hashicorp/terraform-provider-google-beta/v3/google-beta	16.778s
FAIL

```

Reply:
Got it, thanks!